contracts/bonding/BondingManager.sol

```diff
@@ -1319,8 +1363,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     function updateDelegatorWithEarnings(address _delegator, uint256 _endRound, uint256 _lastClaimRound) internal {
         Delegator storage del = delegators[_delegator];
         uint256 startRound = _lastClaimRound.add(1);
-        uint256 currentBondedAmount = del.bondedAmount;
-        uint256 currentFees = del.fees;
+        uint256 currentBondedAmount;
```

Comment:
We should continue to initialize these variables. If a delegator for some reason has a null delegate and then invokes a staking function then its bondedAmount and fees would be set to 0.

Reply:
fixed in 06e8d46